tests/profiles/universal_test.py

```diff
@@ -300,6 +279,90 @@ def test_check_ots():
     assert "Failed to sanitize file!" in message
 
 
+@pytest.mark.parametrize('installed, latest, result', [
+    ("0.5.0",
+     "0.5.0", True),
+    ("0.5.1",
+     "0.5.0", True),
+    ("0.4.1",
+     "0.5.0", False),
+    ("0.3.4",
+     "0.3.5", False),
+    ("1.0.0",
+     "1.0.1", False),
+    ("2.0.0",
+     "1.5.1", True),
+    ("0.5.2.dev73+g8c9ebc0.d20181023",
+     "0.5.1", True),
+    ("0.5.2.dev73+g8c9ebc0.d20181023",
+     "0.5.2", False),
+    ("0.5.2.dev73+g8c9ebc0.d20181023",
+     "0.5.3", False),
+])
+def test_is_up_to_date(installed, latest, result):
+    assert is_up_to_date(installed, latest) is result
+
+
+class MockDistribution:
+    """Helper class to mock pip-api's Distribution class."""
+
+    def __init__(self, version: str):
+        self.name = "fontbakery"
+        self.version = version
+
+    def __repr__(self):
+        return f"<Distribution(name='{self.name}', version='{self.version}')>"
+
+
+# We don't want to make an actual GET request to PyPI.org, so we'll mock it.
```

Comment:
In the rare event of the API changing in a breaking way, we would not be aware of it until it breaks in the hands of our users.

I don't think the current rate of weekly CI runs it so big that it would cause any trouble by flooding the actual PyPI servers with requests.

Reply:
Mocking 3rd-party API requests is a standard practice in unit testing, as it isolates our code from their code. It has nothing to do with not wanting to risk flooding their servers.
I'll add another unit test separate from this one that calls the live API.

Reply:
Added separate test in 26be9be67acb33deb5c581f1562c25f8e84369ec

Reply:
ok, that's good!